src/main/java/nyx/Deadline.java

```diff
@@ -0,0 +1,53 @@
+package nyx;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task with a deadline.
+ * This class inherits from the parent abstract Task class.
+ */
+public class Deadline extends Task {
+    private static final String DATETIME_FORMAT = "yyyy-MM-dd H:m";
+    private final LocalDateTime by;
+
+    /**
+     * Constructs a deadline task with its description, deadline and an indicator of whether it is marked as done.
```

Comment:
The javaDocs convention said that we should leave a gap between the description and @param